README.md

```diff
@@ -1,103 +1,151 @@
-Binary-Static 
-=============
+# Binary-Static
 
 This repository contains the static HTML, Javascript, CSS, and images content of the [Binary.com](http://www.binary.com) website.
 
-## Installation
+![build](https://img.shields.io/circleci/build/github/binary-com/binary-static) ![node](https://img.shields.io/badge/node-%3E%3D10.14.2-blue.svg) ![npm](https://img.shields.io/badge/npm-%3E%3D6.9.0-blue.svg) ![sass](https://img.shields.io/badge/Sass-CC6699?style=flat&logo=sass&logoColor=white)
+**In this document**
 
-In order to work on your own version of the Binary.com Javascript and CSS, please **fork this project**.
+- [Other Documents](#other-documents)
+- [Pre-installation](#pre-installation)
+- [Preview on your local machine](#preview-on-your-local-machine)
+- [Test link deployment](#test-link-deployment)
+- [Manage releases](#manage-releases)
 
-You will also need to install the following on your development machine:
+## Other Documents
+
+- [Scripts](scripts/README.md)
+- [Javascript](src/javascript/README.md)
+- [Sass](src/sass/README.md)
+- [Templates](src/templates/README.md)
+
+## Pre-installation
+
+Before running or contribute to this project, you need to have the setup of the following packages in your environment
 
 - Ruby, RubyGems
 - Sass (`sudo gem install sass`)
-- Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
-- Go to project root, then run `npm install`
+- Node.js (10.14.2 or higher is recommended)
```

Comment:
for node > 12, we have an error for installing node-sass because our node-sass version is not up to date and doesn't support node>12. Do we need to mention this here? if we use node 12, we don't need to install sass separately with gem.  or also we can think about upgrading our sass version